src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -279,11 +279,27 @@ private String getErrorMessage(String service, String existingCaseId, String exc
             return Optional.empty();
         } catch (FeignException.UnprocessableEntity exception) {
             String msg = format(
-                "Service returned 422 Unprocessable Entity response "
+                "CCD returned 422 Unprocessable Entity response "
                     + "when trying to update case for %s jurisdiction "
                     + "with case Id %s "
                     + "based on exception record with Id %s. "
-                    + "Service response: %s",
+                    + "CCD response: %s",
+                exceptionRecord.poBoxJurisdiction,
+                existingCase.getId(),
+                exceptionRecord.id,
+                exception.contentUTF8()
+            );
+            log.error(msg, exception);
+
+            return Optional.of(msg);
+        } catch (FeignException.Conflict exception) {
+            String msg = format(
+                "CCD returned 409 Conflict response "
+                    + "when trying to update case for %s jurisdiction "
+                    + "with case Id %s "
+                    + "based on exception record with Id %s "
+                    + "because it has been updated in the meantime. "
```

Comment:
this is just our assumption. Perhaps this can be removed and building this message could be them moved to a shared function.

Reply:
This message will go to the caseworker, so the message returned by this method should be simple, e.g.
```
This action couldn't be completed, because this exception record was being updated by someone else.
```
Ideally, message should be agreed with Ade.

Reply:
It's ok to log an error like this, though.